### PR TITLE
v1.1.x

### PR DESCRIPTION
**Pull request by generall**

### CODE REVIEW

File: qdrant/v1.1.x/quantization.md [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+            ignore=False,
+            rescore=True,
+        )
+    )
+)
+```
+
+`ignore` - whether to ignore quantized vectors during the search process. By default, Qdrant will use quantized vectors if they are available.
+
+`rescore` - Having the original vectors available, Qdrant can re-svaluate top-k search results using the original vectors.
```
Comment (agourlay) [inline qdrant/v1.1.x/quantization.md:159]:
```suggestion
`rescore` - Having the original vectors available, Qdrant can re-evaluate top-k search results using the original vectors. 
```

File: qdrant/v1.1.x/quantization.md [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+- **enable rescore**: Having the original vectors available, Qdrant can re-evaluate top-k search results using the original vectors. On large collections, this can improve the search quality, with just minor performance impact.
+
+
+#### Memory and speed tuning
+
+In this section, we will discuss how to tune the memory and speed of the search process with quantization.
+
+There are 3 possible modes to place storage of vectors within the qdrant collection:
+
+- **All in RAM** - all vector, original and quantized, are stored in RAM. This is the fastest mode, but requires a lot of RAM. Enabled by default.
```
Comment (agourlay) [inline qdrant/v1.1.x/quantization.md:219]:
```suggestion
- **All in RAM** - all vector, original and quantized, are loaded and kept in RAM. This is the fastest mode, but requires a lot of RAM. Enabled by default.
```

File: qdrant/v1.1.x/quantization.md [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+    quantization_config=models.ScalarQuantization(
+        scalar=models.ScalarQuantizationConfig(
+            type=models.ScalarType.INT8,
+            always_ram=True,
+        ),
+    ),
+)
+```
+
+In this scenario, number of disk reads may play a significant role in the search speed.
```
Comment (agourlay) [inline qdrant/v1.1.x/quantization.md:264]:
```suggestion
In this scenario, the number of disk reads may play a significant role in the search speed.
```

**Review by agourlay — COMMENTED** [review]
Went through the quantization doc, great work :+1: 

My only issue is the confusion regarding `storing quantized vectors` in RAM.

It sounds like they are ephemeral whereas there are actually persisted on disk but loaded and kept in RAM.

File: qdrant/v1.1.x/quantization.md [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+
+`quantile` - the quantile of the quantized vector components.
+The quantile is used to calculate the quantization bounds.
+For instance, if you specify `0.99` as the quantile, 1% of extreme values will be excluded from the quantization bounds.
+
+Using quantiles lower than `1.0` might be useful if there are outliers in your vector components.
+This parameter only affects the resulting precision and not the memory footprint.
+It might be worth tuning this parameter if you experience a significant decrease in search quality.
+
+`always_ram` - whether to store quantized vectors in RAM or on disk. By default, quantized vectors are stored on the same place as the original vectors.
```
[the remark this answers is not in the document]
Comment (generall) [reply qdrant/v1.1.x/quantization.md:112]: `store` -> `keep` ?

[the remark this answers is not in the document]
Comment (generall) [reply qdrant/v1.1.x/quantization.md]: > `always_ram` - whether to keep quantized vectors always cached in RAM or not. By default, quantized vectors are loaded in the same way as the original vectors.

File: qdrant/v1.1.x/quantization.md [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+There are tradeoffs associated with quantization.
+On the one hand, quantization allows for significant reductions in storage requirements and faster search times.
+This can be particularly beneficial in large-scale applications where minimizing the use of resources is a top priority.
+On the other hand, quantization introduces an approximation error, which can lead to a slight decrease in search quality.
+The level of this tradeoff depends on the quantization method and its parameters, as well as the characteristics of the data.
+
+
+## Scalar Quantization
+
+*Avaliable in Qdrant since v1.1.0*
```
Comment (joein) [inline qdrant/v1.1.x/quantization.md:26]:
```suggestion
*Available in Qdrant since v1.1.0*
```

File: qdrant/v1.1.x/quantization.md [hunk 6]
```diff
@@ -0,0 +1,344 @@
...
+
+Scalar quantization, in the context of vector search engines, is a compression technique that compresses vectors by reducing the number of bits used to represent each vector component.
+
+
+For instance, Qdrant uses 32-bit floating numbers to represent the original vector components. Scalar quantization allows you to reduce the number of bits used 8.
```
Comment (joein) [inline qdrant/v1.1.x/quantization.md:31]:
```suggestion
For instance, Qdrant uses 32-bit floating numbers to represent the original vector components. Scalar quantization allows you to reduce the number of bits used to 8.
```

**Pull request merged**